tests/integration/test_run_app_from_bundle.py

```diff
@@ -0,0 +1,53 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from os import path
+import subprocess
+import tarfile
+
+import utils
+
+
+def test_run_app_from_bundle(test_env, default_request, tmpdir):
```

Comment:
It'd be nice to skip this test if `go` is not installed on the system:
```suggestion
@pytest.mark.skipif(not shutil.which("go"), reason="requires go to be installed")
def test_run_app_from_bundle(test_env, default_request, tmpdir):
```